src/bundles/csg/functions.ts

```diff
@@ -181,6 +178,15 @@ export const white: string = '#FFFFFF';
 
 // [Functions - Primitives]
 
+/**
+ * An empty Shape.
+ *
+ * @category Primitives
+ */
+export function empty_shape(): Shape {
+  return new Shape();
+}
+
```

Comment:
Why is `empty_shape` a nullary function? Why can't it be just a shape value? I think it should be just called `empty`, and not `empty_shape`. Also, cube is just `cube` and not `cube_shape`.

Reply:
Exported a function here to standardize with other primitives, which are all functions. I was feeling that `empty` probably doesn't provide that much context to the readers as to what this is?

Reply:
I'm ok with calling it `empty_shape` but it should be a shape value, not a function. This would be similar to the runes `heart`, `square` etc in the `rune` module. BTW, there we call it `blank`. Should we call it `blank` here too?

Reply:
The primitive shapes used to all be values, but changes to the way users specified colours meant they all became functions, e.g. `cube(cyan)`. Therefore all currently provided shapes are functions:

![image](https://github.com/source-academy/modules/assets/44526554/4fdb39f5-969f-4b14-9236-203b77af7010)

As an empty shape wouldn't have a colour, it is true it does not need to be a function, but for consistency perhaps it would be neater to keep it as a nullary function. The only values CSG currently provides are colour strings, so if we were to export an `empty` value, it would be "on its own" as a shape (in its own category in the [documentation](https://source-academy.github.io/modules/documentation/modules/csg.html)).

As for naming, I'm personally leaning towards `empty` over `empty_shape`, and `blank` feels like it better fits runes over shapes.

Reply:
The situation in `csg` is quite similar to `rune`. We have primitives (including an empty special case) and a way to color things. The two libraries go different routes in this. I feel that consistency between the two is more important than which way we go. So I'll approve this one, and make a separate issue regarding consistency between `rune` and `csg`.